app/Services/Email.php

```diff
@@ -133,8 +134,6 @@ protected function parseLinks($string)
      */
     protected function setupEmail()
     {
-        $leaderboardVars = $this->getLeaderboardVars($this->users);
```

Comment:
I think this was moved out of the loop on purpose, since the leaderboard is the same for all users